frontend/testing/requests/droplet.test.js

```
@@ -0,0 +1,503 @@
+  ]);
+});
+
+describe("Droplet API Functions", () => {
+  describe("getDroplets", () => {
+    it("successfully fetches droplets with default parameters", async () => {
+      const mockDroplets = [
+        { id: 1, name: "Droplet 1" },
+        { id: 2, name: "Droplet 2" },
+      ];
+
+      const { fetchAPI } = require("@/lib/utils");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)